README.md

```diff
@@ -24,14 +24,33 @@ See [splinter.dev](https://www.splinter.dev/) to learn about Splinter.
 
 ## Building Splinter
 
-To build Splinter, run `cargo build` from the root directory. This command
-builds all of the Splinter components, including `libsplinter` (the main
-library), `splinterd` (the splinter daemon), the CLI, the client, and all
-examples in the `examples` directory.
+Splinter is built using latest stable [rust](https://www.rust-lang.org/), which
+you should install via [rustup](https://rustup.rs/).
+
+To install the remaining dependencies using a package manager, run one of the
+following commands.
+
+Homebrew (OS X):
+```bash
+brew install openssl zeromq pkg-config protobuf postgresql
```

Comment:
Don't you need to specify that the openssl installed via brew is available on a specific path?